libobs/obs.h

```diff
@@ -247,10 +246,8 @@ struct obs_source_frame {
 	bool full_range;
 	float color_range_min[3];
 	float color_range_max[3];
-	union {
-		uint8_t flip : 1; /* deprecated */
-		uint8_t flags;
-	};
+	bool flip;
+	uint8_t flags;
```

Comment:
This will break backwards compatibility, as the `union` allocated a uint8_t that is now moved away from the original location. In order to maintain support for older plugins/integrations, the location of `bool flip` and `uint8_t flags` should be swapped.

Reply:
I moved the linear alpha flag location, which hasn't seen a release. The flip bit position is consistent.

Original:
flip (0000000_), 3-byte padding

Unreleased change:
union: flip (0000000_) & linear alpha (000000_0), 3-byte padding

Current PR:
flip (0000000_), flags (0000000_), 2-byte padding